polaris-react/src/components/Avatar/Avatar.tsx

```diff
@@ -96,6 +104,7 @@ export function Avatar({
     size && styles[variationName('size', size)],
     !customer && styles[variationName('style', styleClass(nameString))],
     hasImage && status === Status.Loaded && styles.imageHasLoaded,
+    shape === 'square' && styles.SquareAvatar,
```

Comment:
I'd use the variationName util here for consistency

```suggestion
    shape && styles[variationName('shape', shape)],
```

You'll have to change the class name to match